test/system/520-checkpoint.bats

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# test podman checkpoint. Similar in many ways to our pause tests.
+#
+
+load helpers
+
+CHECKED_ROOTLESS=
+function setup() {
+    # FIXME: https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1857257
+    # TL;DR they keep fixing it then breaking it again. There's a test we
+    # could run to see if it's fixed, but it's way too complicated. Since
+    # integration tests also skip checkpoint tests on Ubuntu, do the same here.
+    if grep -qiw ubuntu /etc/os-release; then
+        skip "FIXME: checkpointing broken in Ubuntu 2004, 2104, 2110, ..."
+    fi
+
+    # None of these tests work rootless....
+    if is_rootless; then
+        # ...however, is that a genuine cast-in-stone limitation, or one
+        # that can some day be fixed? If one day some PR removes that
+        # restriction, fail loudly here, so the developer can enable tests.
```

Comment:
There is a pull request that aims to enable rootless checkpoint/restore (https://github.com/checkpoint-restore/criu/pull/1155).
It is still work in progress, but the `CAP_CHECKPOINT_RESTORE` has been merged.